dashboard-backend/main.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+import (
+	"github.com/Aman-Codes/e2e-dashboard-backend/pkg/router"
```

Comment:
we can import using litmuschaos
```suggestion
	"github.com/litmuschaos/e2e-dashboard-backend/pkg/router"
```